authentik/sources/ldap/models.py

```diff
@@ -20,6 +20,7 @@
 from authentik.lib.models import DomainlessURLValidator
 
 LDAP_TIMEOUT = 15
+LDAP_DISTINGUISHED_NAME = "distinguishedName"
```

Comment:
I needed to move this here to avoid a cyclical dependency, and since models is fairly self-sufficient from within `ldap/` package, I thought it a safe and sensible place to move it to